src/cloud/reducers/limits.ts

```diff
@@ -102,8 +104,9 @@ export const limitsReducer = (
         }
 
         if (limits.rate) {
-          const {readKBs, writeKBs, cardinality} = limits.rate
+          const {queryTime, readKBs, writeKBs, cardinality} = limits.rate
 
+          draftState.rate.queryTime.maxAllowed = queryTime / 1e9
```

Comment:
Another instance of modeling after another line of code that addresses a large number (line70). I just assumed that was the preferred method for handling large numbers in the UI code. If it is serving no purpose, I can remove it.

Reply:
@gspletzer representing `1000000000` as `1e9` is completely fine. I was commenting more on the idea that either representation is a magic number, that is a number without context. What does the number one billion represent in this case? E.g. it's the total amount of query time in milliseconds that a free account gets.

Looking back over the code, I don't think it's as big of a deal since it's being computed to be stored into a variable that does provide context, `maxAllowed`. So in summary, I think it's okay to ignore this comment.